applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -326,6 +328,8 @@ def AssembleTestSuites():
 
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestAdjointSensitivityAnalysisBeamStructure]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestAdjointSensitivityAnalysisShell3D3NStructure]))
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestAdjointSensitivityAnalysisLinearTrussStructure]))
```

Comment:
Test are always great, but if it is a full case then consider adding it to the nightly tests

The samllsuite already takes too long (and therefore crashes Travis sometimes)

I am planning to soon move the too slow tests to the nightly suite anyway

Reply:
I added the new cases to the nightly tests